walkthroughs/howto-access-log/README.md

```diff
@@ -0,0 +1,174 @@
+## Access Log Format Feature Background
+Today App Mesh supports configuring access log file path (https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy-logs.html) for virtual nodes (https://docs.aws.amazon.com/app-mesh/latest/userguide/virtual_nodes.html) and virtual gateways (https://docs.aws.amazon.com/app-mesh/latest/userguide/virtual_gateways.html). We apply the same configuration for all the listener filter chains in Envoy. If not specified, Envoy will output logs to /dev/stdout by default. 
```

Comment:
I think maybe this background should be rewritten, it seems targeted at folks testing an unreleased feature, talking about the current state of the system an adding a new feature, while for the audience of this example the logging format is an existing feature they want to learn about

Reply:
Will do